src/main/java/Main.java

```diff
@@ -0,0 +1,8 @@
+import duke.*;
```

Comment:
It seems unclear which classes of the `duke` package is required for the main application, you might want to consider explicit imports.